spec/menu_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'Menu'
+
+describe Menu do
+
+  subject(:menu) {described_class.new}
+  let(:takeaway){double :takeaway, :view_menu => menu.menu_layout}
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)